mucommander-os-api/src/main/java/com/mucommander/desktop/ActionShortcuts.java

```diff
@@ -284,6 +284,10 @@ public KeyStroke getDefaultAltKeyStroke(ActionType actionId) {
             return KeyStroke.getKeyStroke(KeyEvent.VK_LEFT, 0);
         case GoToRoot:
             return KeyStroke.getKeyStroke(KeyEvent.VK_BACK_SLASH, KeyEvent.CTRL_DOWN_MASK);
+        case MarkExtension:
+            return KeyStroke.getKeyStroke(KeyEvent.VK_EQUALS, 0);
```

Comment:
wouldn't this mean that we won't be able to trigger quick-search by pressing '='? same for '-' that is handled below

Reply:
by quick-search you mean moving the cursor when you start typing its name?
If so, then yes, those '=' and '-' (and '+') won't work for quick search initial character. Additional remark, making non-numpad '+' and '-' work as their numpad counter-parts will make it more consistent (especially than in muC main menu under `Mark` you can see '+' and '-' for Mark/Unmark actions and you wonder why they don't work if you don't have numpad /I guess pretty common nowadays on laptops/)

Reply:
> by quick-search you mean moving the cursor when you start typing its name? If so, then yes, those '=' and '-' (and '+') won't work for quick search initial character.

yep, that's what I meant - so it would be better to choose alternative shortcuts that don't limit this functionality

> Additional remark, making non-numpad '+' and '-' work as their numpad counter-parts will make it more consistent (especially than in muC main menu under `Mark` you can see '+' and '-' for Mark/Unmark actions

yeah, that depends on the operating system - we saw that weird representation of numpad in macOS which may confuse users but in fedora I believe it's clearer as, e.g., it's rendered as "NumPad +" for "Mark files..."

> and you wonder why they don't work if you don't have numpad\ /I guess pretty common nowadays on laptops/)

right, and not only on laptops - for instance, I use the Apple Magic Keyboard that doesn't have numpad. I agree we should avoid having actions that are only correlated with shortcuts that include numpad keys

Reply:
> yep, that's what I meant - so it would be better to choose alternative shortcuts that don't limit this functionality

I haven't seen in my life a file or folder with leading = or +, and having a different shortcut without +/=/- won't be intuitive. If somebody wanted to quick search for such exotic filenames then they can remove such shortcuts from defaults.

> > Additional remark, making non-numpad '+' and '-' work as their numpad counter-parts will make it more consistent (especially than in muC main menu under `Mark` you can see '+' and '-' for Mark/Unmark actions
> 
> yeah, that depends on the operating system - we saw that weird representation of numpad in macOS which may confuse users but on fedora I believe it's clearer as, e.g., the shortcut of "Mark files..." is rendered as "NumPad +"

This doesn't solve problem for ppl not having numpads :)

> 
> > and you wonder why they don't work if you don't have numpad\ /I guess pretty common nowadays on laptops/)
> 
> right, and not only on laptops - for instance, I use the Apple Magic Keyboard that doesn't have numeric keypad. I agree we should avoid having actions that are only correlated with shortcuts that include numpad keys

Let's figure out something then :)

Reply:
@pskowronek the way quick-search works doesn't force to match the filename right from the start, I can type "-tmp" and all folders that match ".*-tmpdir.*" would match, so the best practice would be not to use keys that can be part of filenames as shortcuts - how about adding 'ALT' to all those shortcuts?

Reply:
Tried with ALT:

1) ALT+Shift+=  produces this letter on mac: https://en.wiktionary.org/wiki/%C4%AB - as it is a letter, then quick search is engaged 
2) ALT+= - it looks like single file selection is engaged

To avoid the mess, I've tried with CTRL - no issues on mac. I will provide update in a minute (plus Ctrl+Shift+8 -> * i.e. invert selection). Pls tell me what you think.

Reply:
looks good, did you check that those shortcuts don't conflict with existing ones?

Reply:
checked if sth reacts to those shortcuts and also whether if they conflict if existing ones by manually adding them in Preferences - nothing came up.